sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/JsonSupport.scala

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.scalasdk
+
+import com.google.protobuf.any.{ Any => ScalaPbAny }
+import com.akkaserverless.javasdk.{ JsonSupport => JavaJsonSupport }
+
+import scala.jdk.OptionConverters._
+import scala.reflect.ClassTag
+
+object JsonSupport {
+
+  /**
+   * Encode the given value as JSON using Jackson and put the encoded string as bytes in a protobuf Any with the type
+   * URL {@code "json.akkaserverless.com/[valueClassName]"}.
+   *
+   * <p>Note that if the serialized Any is published to a pub/sub topic that is consumed by an external service using
+   * the class name suffix this introduces coupling as the internal class name of this service becomes known to the
+   * outside of the service (and for exampe renaming it may break existing consumers). For such cases consider using the
+   * overload with an explicit name for the JSON type instead.
+   *
+   * @see
+   *   {{encodeJson(T, String}}
```

Comment:
```suggestion
   *   {{encodeJson(T, String)}}
```

or 

```suggestion
   *   {{{encodeJson(T, String)}}}
```

Now you made me doubt, should we use `{{{` or `{{`? I thought it was `{{{`, but may be wrong.

Reply:
I think it was IntelliJ "transforming" the Java to Scala when I copied the docs and method signatures doing this without me noticing. I'll go over it another round.